README.md

```diff
@@ -628,6 +628,15 @@ secure_variables:
   MY_PASSWORD: "s&Du#@$xsSa"
 ```
 
+> There is no easy way to generate obtain [encrypted value](https://github.com/tomzo/gocd-yaml-config-plugin/issues/5) from GoServer, alternatively you can login into go-server and execute the following command to generate encrypted value
+
+```sh
+sudo su go
+cd /etc/go
+# Copy the value inside your cipher file and use it in place of YOUR-CIPHER-FILE-VALUE
+echo -n 'YOUR-INPUT' | openssl enc -des-cbc -a -iv 0 -K YOUR-CIPHER-FILE-VALUE
```

Comment:
@jskswamy Thank you for the PR, I was going to add this. Can we make it into a single line?
```
sudo -u go bash -c 'echo -n 'YOUR-INPUT' | openssl enc -des-cbc -a -iv 0 -K $(cat /etc/go/cipher)'
```